codes/quantum/qubits/lifted_product.yml

```diff
@@ -7,30 +7,35 @@ code_id: lifted_product
 physical: qubits
```

Comment:
Please merge with existing entry for lifted_product, which was provided to me by one of the authors (Pavel Panteleev).